apps/happy-blocks/src/pricing-plans/components/billing-info.tsx

```diff
@@ -0,0 +1,21 @@
+import { FunctionComponent } from 'react';
+import { PricingPlanBilling } from '../types';
+
+interface Props {
+	billing: PricingPlanBilling;
+}
+
+const BillingInfo: FunctionComponent< Props > = ( { billing } ) => {
+	return (
+		<div className="wp-block-a8c-pricing-plans__detail-current-billing">
```

Comment:
Nit: perhaps using `billing-info` instead of `detail-current-billing` would be more suitable here?